src/Card.js

```diff
@@ -3,9 +3,17 @@ import PropTypes from 'prop-types'
 import ReactDOM, {unstable_renderSubtreeIntoContainer as renderSubtreeIntoContainer} from 'react-dom'
 import assign from 'object-assign'
 
-class Card extends React.Component {
-  static propTypes = {
+
+const defaultProps = {
+    active: false,
+    position: 'right',
+    arrow: null,
+    style: {style: {}, arrowStyle: {}}
+};
+
+const propTypes = {
     active: PropTypes.bool,
+    parentEl: PropTypes.object,
```

Comment:
Since `parentEl` is required in the outer component, maybe we should mark it as required here as well?

Reply:
yes indeed